cli_reference/openshift_cli/usage-oc-kubectl.adoc

```diff
@@ -21,6 +21,44 @@ The `oc` binary offers a built-in `login` command that allows authentication and
 +
 The additional command `oc new-app`, for example, makes it easier to get new applications started using existing source code or pre-built images. Similarly, the additional command `oc new-project` makes it easier to start a project that you can switch to as your default.
 
+The {product-title} version must match between master and node hosts, excluding
+temporary mismatches during cluster upgrades. For example, in a
+{product-version} cluster, all masters must be {product-version} and all nodes
+must be {product-version}. If you installed an earlier version of `oc`, you
+cannot use it to complete all of the commands in {product-title}
+{product-version}. You must download and install the new version of `oc`.
```

Comment:
Should this following sentence be bold? It's a fairly strongly worded statement.

Reply:
OCP docs guidelines mostly shy away from bolding/italicizing sentences. There are rare cases when I can use italics, but not sure if it's warranted here. If you're strongly in favor of italics I'm happy to see how the PR squad responds to it :)